mbox/src/test/java/org/apache/james/mime4j/mboxiterator/ByteBufferInputStreamTest.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.james.mime4j.mboxiterator;
```

Comment:
We are missing the compulsory Apache license header. Can you add it like in the other files?

Reply:
Done.